python/lsst/atmospec/centroiding.py

```diff
@@ -77,9 +78,9 @@ class SingleStarCentroidTaskConfig(pipeBase.PipelineTaskConfig,
         target=AstrometryTask,
         doc="Task to perform astrometric calibration to refine the WCS",
     )
-    qfmTask = pexConfig.ConfigurableField(
-        target=QuickFrameMeasurementTask,
-        doc="XXX",
+    centroidingFallbackTask = pexConfig.ConfigurableField(
+        target=PeekExposureTask,
+        doc="The task to run find the brightest star if astrometry fails",
```

Comment:
This is technically an API change and should go through the deprecation procedure. Are you skipping this because this has not been used extensively (in production) and hence okay to just switch? I'm guessing as much from the previous doc 😆

Reply:
Haha, yeah, exactly. I personally know everyone who's using this in the wild. It's basically just Jérémy.